instance/instance_external_test.go

```diff
@@ -566,9 +567,52 @@ func TestStore_UpdateImportTask_UpdateImportObservations(t *testing.T) {
 	})
 }
 
-func TestStore_UpdateImportTask_UpdateImportObservations_InvalidState(t *testing.T) {
+func TestStore_UpdateImportTask_UpdateImportObservations_Failure(t *testing.T) {
 
 	t.Parallel()
+	Convey("update to an import task with invalid json returns http 400 response", t, func() {
+		body := strings.NewReader(`{`)
+		r := createRequestWithToken("PUT", "http://localhost:21800/instances/123/import_tasks", body)
+		w := httptest.NewRecorder()
+
+		mockedDataStore := &storetest.StorerMock{
+			UpdateImportObservationsTaskStateFunc: func(id string, state string) error {
+				return nil
+			},
+		}
+
+		instance := &instance.Store{Host: host, Storer: mockedDataStore}
```

Comment:
Minor point - by calling this var instance you are shadowing the `instance` package, the test is in `instance_test` package so you don't have access to vars/constants etc in that package.